lastuser_core/models/user.py

```diff
@@ -171,6 +171,14 @@ def add_email(self, email, primary=False, type=None, private=False):
             self.primary_email = useremail
         return useremail
 
+    def make_email_primary(self, email):
```

Comment:
This method doesn't need to exist. `self.primary_email` is already safe to assign to. Always load the `UserEmail` object first if you don't have it.